source/lac/petsc_vector_base.cc

```diff
@@ -155,6 +155,7 @@ namespace PETScWrappers
     , ghosted(false)
     , last_action(::dealii::VectorOperation::unknown)
   {
+    /* TODO GHOSTED */
```

Comment:
Can you elaborate what the TODO refers to? Right now you're probably the only one who actually knows what it means :-)

Reply:
Added a note here https://github.com/dealii/dealii/issues/14656

Reply:
Can you put your comment above into a comment in the code? This way all of the information needed to address the TODO is in the code.

Reply:
will do

Reply:
@bangerth I have added support here https://github.com/dealii/dealii/pull/14481/commits/acf141cc565cb6b4bdc34f56098fbba2e2f18b0c